app/models/game.rb

```diff
@@ -5,16 +5,6 @@ class Game < ActiveRecord::Base
 
 	after_create :populate_board!
 
-
-	def index
-		@list_all_games=Game.all
```

Comment:
The code should not never have errors when merging into master.  It should never break existing functionality.  However, you can push unfinished code out so that everyone gets it.

e.g. If you are working on a branch that includes a database migration.  It's possible you may decide to create a PR and merge the migration before the feature is finished if you'd like it to show up in master, especially if someone else has information that will touch that migration.